de/news/_posts/2017-04-14-deutsche-ruby-mailingliste.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news_post
+title: "Deutschsprachiges Ruby-Forum schließt und wird durch Mailingliste ersetzt"
```

Comment:
(Would come across more like an invitation to _all_ Rubyists instead of being "only" a replacement.)